x/claim/keeper/msg_server_claim_eth.go

```diff
@@ -57,13 +57,28 @@ func (k msgServer) ClaimEth(goCtx context.Context, msg *types.MsgClaimEth) (*typ
 		Address:                msg.Creator,
 		Chain:                  types.ARKEO,
 		InitialClaimableAmount: ethClaim.InitialClaimableAmount,
-		ActionCompleted:        []bool{false, false},
+		ActionCompleted:        []bool{true, false, false}, // true so they do not make them claim again on arkeo.
 	}
+
+	ctx.EventManager().EmitEvents(sdk.Events{
+		sdk.NewEvent(
+			types.EventTypeClaimFromEth,
+			sdk.NewAttribute(sdk.AttributeKeySender, strings.ToLower(msg.EthAddress)),
+			sdk.NewAttribute(sdk.AttributeKeyAmount, ethClaim.InitialClaimableAmount.String()),
+		),
+	})
+
 	err = k.SetClaimRecord(ctx, arkeoClaim)
 	if err != nil {
 		return nil, errors.Wrapf(err, "failed to set claim record for %s", msg.Creator)
 	}
 
+	// call claim on arkeo to claim arkeo
+	_, err = k.ClaimCoinsForAction(ctx, msg.Creator, types.ACTION_CLAIM)
```

Comment:
I think [the](https://github.com/arkeonetwork/arkeo/pull/41/files#diff-2e39f49e8b029fe4d783197ca9d2cf7f4abcf97a9ec1188a76e2d820e2a57dbdR60) , it should be `false`
However , in `ClaimCoinsForAction` we should double check whether the action has been claimed or not

Reply:
it is false now and we do check in ClaimCoinsForAction